File: desafio-proway-2/BackEnd/Training_API/Models/TrainingRoomPerson.cs
namespace Training_API.Models
{
    public class TrainingRoomPerson
    {
        public TrainingRoomPerson(int id, int stageTimeId, int trainingRoomId, int personId)
        {
            this.Id = id;
            this.StageTimeId = stageTimeId;
            this.TrainingRoomId = trainingRoomId;
            this.PersonId = personId;
        }
        public int Id { get; set; }
        public int StageTimeId { get; set; }
        public StageTime StageTime { get; set; }
        public int TrainingRoomId { get; set; }
        public TrainingRoom TrainingRoom { get; set; }
        public int PersonId { get; set; }
        public Person Person { get; set; }
    }
}

File: desafio-proway-2/FrontEnd/Training/src/app/models/TrainingRoomPerson.ts
import { Person } from './Person';
import { StageTime } from "./StageTime";
import { TrainingRoom } from "./TrainingRoom";

export class TrainingRoomPerson {
  
  constructor() {
    this.id = 0;
    this.stageTimeId = 0;
    this.stageTime = new StageTime();
    this.trainingRoomId = 0;
    this.trainingRoom = new TrainingRoom();
    this.personId = 0;
    this.person = new Person();
  }
  
  id: number;
  stageTimeId: number;
  stageTime: StageTime;
  trainingRoomId: number;
  trainingRoom: TrainingRoom;
  personId: number;
  person: Person;
}
